addons/waterways/river_gizmo.gd

```diff
@@ -213,17 +422,42 @@ func commit_handle(gizmo: EditorSpatialGizmo, index: int, restore, cancel: bool
 	ur.add_undo_property(river, "valid_flowmap", river.valid_flowmap)
 	ur.add_undo_method(river, "update_configuration_warning")
 	ur.commit_action()
-	
+
+	_extra_lines = {}
+
 	redraw(gizmo)
 
+func draw_vertical_line_to_terrain(index, pos, global_pos, space_state, material):
```

Comment:
This is pseudo private, so it should have a leading "_", also I'd like the arguments to be typed. I know the lines get long, but I still thinks it's preferable.